packages/runtime/src/account.ts

```diff
@@ -144,6 +150,30 @@ export class StoreAccount implements StoreAccountI {
     return this.appsLocalState.get(appId);
   }
 
+  /**
+   * Returns asset definitions
+   * @param assetId asset index
+   */
+  getAssetDef (assetId: number): AssetDef | undefined {
+    return this.createdAssets.get(assetId);
+  }
+
+  /**
+   * Creates Asset in account's state
+   * @param name Asset Name
+   * @param asaDef Asset Definitions
+   */
+  createAsset (assetId: number, name: string, asaDef: ASADef, creator: string): AssetDef {
```

Comment:
We don't need to pass the `creator` here, we are already inside an account. You can use `this.address` instead of passing creator explicitly